var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -45,6 +45,8 @@ class Gdb(AutotoolsPackage, GNUMirrorPackage):
     variant('ld', default=False, description='Enable ld')
     variant('tui', default=False, description='Enable tui')
 
+    patch('gdb-libintl.patch', level=0, when='@10.1')
```

Comment:
Can you add a brief comment on why the patch is needed, if there's an issue or a bug fix upstream etc.?

Reply:
Basically, without it, I get a bunch of undefined references to libintl_gettext on CentOS 8, while linking gdbserver.
https://www.gnu.org/software/gettext/FAQ.html#integrating_undefined

I don't have any documentation nor I saw that the problem was reported with the gdb community.
I principle I could report it to them too.

Reply:
Would you like to have it on the top of the package file?

Reply:
A brief comment above the patch directive that includes the link is fine. I encountered the same issue in #22382 so that was kind of clear in the diff during review, but having a brief sentence above the patch will help the next reader to reason about what's happening during build.

Reply:
Done